doc/3/controllers/document/m-update/index.md

```diff
@@ -0,0 +1,82 @@
+---
+code: true
+type: page
+title: mUpdate
+description: Updates multiple documents
+---
+
+# mUpdate
+
+Updates multiple documents.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents)
+throws NotConnectedException, InternalException
+
+public CompletableFuture<ConcurrentHashMap<String, ArrayList<Object>>> mUpdate(
+      final String index,
+      final String collection,
+      final ArrayList<ConcurrentHashMap<String, Object>> documents,
+      final UpdateOptions options)
+throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                                    | Description                       |
+| ------------------ | ------------------------------------------------------- | --------------------------------- |
+| `index`            | <pre>String</pre>                                       | Index                             |
+| `collection`       | <pre>String</pre>                                       | Collection                        |
+| `documents`        | <pre>ArrayList<ConcurrentHashMap<String, Object>></pre> | ArrayList containing the documents to update |
+| `options`          | <pre>UpdateOptions</pre><br>(`null`)                    | Query options |
+
+---
+
+### documents
+
+Each document has the following properties:
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `_id`              | <pre>String</pre>                            | Optional document ID. Will be auto-generated if not defined.             |
```

Comment:
Seems to be copy-pasted from the document:create documentation: document identifiers are not optional for update methods.

Reply:
In deed